quinn-udp/src/lib.rs

```diff
@@ -115,6 +115,13 @@ pub struct RecvMeta {
     /// Populated on platforms: Windows, Linux, Android (API level > 25),
     /// FreeBSD, OpenBSD, NetBSD, macOS, and iOS.
     pub dst_ip: Option<IpAddr>,
+    /// A timestamp for when the given packet was received by the operating system.
+    /// Timestamping must be enabled for this information to be collected.
+    ///
+    /// Populated on platforms with clock sources:
+    /// - Linux: CLOCK_REALTIME (see docs for SO_TIMESTAMP)
+    #[cfg(not(wasm_browser))]
```

Comment:
Can we just make this `cfg(os = "linux")` only for now? Not much point in making this available as `None`-only on platforms where we don't have support for it.